lib/slack-ruby-bot/commands/base.rb

```diff
@@ -22,6 +22,17 @@ def self.send_message_with_gif(client, channel, text, keywords, options = {})
         send_message client, channel, text, options
       end
 
+      def self.send_gif(client, channel, keywords, options = {})
+        gif = begin
+          Giphy.random(keywords)
+        rescue StandardError => e
+          logger.warn "Giphy.random: #{e.message}"
+          nil
+        end
+        text = gif.image_url.to_s if gif
+        send_message client, channel, text, options
```

Comment:
There's a lot of duplication in this code, looks very similar to `send_message_with_gif`, I would write one in terms of the other.